apps/web/src/components/editor/tiptap.tsx

```diff
@@ -443,14 +444,36 @@ function TiptapWrapper(
     }
   }, [isHydrating]);
 
+  useEffect(() => {
+    if (!editorContainerRef.current) return;
+
+    const handleWheel = (e: WheelEvent) => {
+      if (e.ctrlKey) {
+        e.preventDefault();
+        const delta = e.deltaY;
+        let zoom = editorConfig.zoom;
+        zoom += Math.round(delta);
```

Comment:
@thecodrr I think for the mouse scroll event, we should give user the control on what zoom level they want to be instead of increasing/decreasing by 10 ourselves.

Reply:
Agreed